ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -572,6 +610,27 @@ tests:
       IP_STACK: v4v6
       NUM_EXTRA_WORKERS: "3"
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-ipv4v6-3masters-ocp-410
+  optional: true
+  run_if_changed: ^(deploy/operator/.*)$
```

Comment:
please update the corresponding 4.9 job to not run always

Reply:
Ok, disabled